setup.py

```diff
@@ -200,6 +200,7 @@ def do_setup():
         scripts=['airflow/bin/airflow'],
         install_requires=[
             'alembic>=0.8.3, <0.9',
+            'blinker==1.4',
```

Comment:
is it a apache license compatible library? Are you going to open source the pr?

Reply:
This should be find as blinker is MIT, which is in category a of ASF approved license.

https://www.apache.org/legal/resolved.html#category-a